foundation/foundation-upgrade/src/test/resources/test-files/v1_7_0/FoundationCoreJavaMigration/migration/Metadata.java

```diff
@@ -0,0 +1,64 @@
+package org.boozallen.aissemble.cdi;
+
+/*-
+ * #%L
+ * sparkpipeline2::Pipelines::Spark Pipeline
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.jboss.weld.environment.se.WeldContainer;
+
+import com.boozallen.aiops.core.metadata.MetadataAPI;
```

Comment:
Change to aissemble.core(?)

Reply:
These files have been removed since we moved to a single scenario to cover these test cases.